tests/test_wave_03.py

```diff
@@ -55,12 +55,12 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
-    # *************************************************************************************************
-    # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
-    # **************************************************************************************************
+    #raise Exception("Test needs to be completed.")
+    assert friends_unique_movies[0] != friends_unique_movies[1]
+    assert friends_unique_movies[0] != friends_unique_movies[2]
+    assert friends_unique_movies[1] != friends_unique_movies[2]
```

Comment:
This set of asserts ensures that all of the movies are unique, but it doesn't check to make sure the correct movies are in the list.  I recommend adding a set of asserts that verifies the contents of the list.
```suggestion
    assert friends_unique_movies[0] != friends_unique_movies[1]
    assert friends_unique_movies[0] != friends_unique_movies[2]
    assert friends_unique_movies[1] != friends_unique_movies[2]
    assert INTRIGUE_3 in friends_unique_movies
    assert HORROR_1 in friends_unique_movies
    assert FANTASY_4 in friends_unique_movies
```